CMake/resolve_dependency_modules/README.md

```diff
@@ -1,7 +1,47 @@
+# Dependency List
+Following is the list of libraries and their minimum version
+that Velox requires. Some of these libraries can be installed
+via a platform's package manager (eg. `brew` on MacOS).
+The versions of certain libraries is the default provided by 
+the platform's package manager. Some libraries can be bundled
+by Velox. See details on bundling below.
+
+| Library Name      | Minimum Version | Bundled? | 
+|-------------------|-----------------|----------|
+| ninja             | default         | No       |
+| ccache            | default         | No       |
+| icu4c             | default         | Yes      |
+| gflags            | default         | yes      |
+| glog              | default         | Yes      |
+| gtest (testing)   | default         | Yes      |
+| libevent          | default         | No       |
+| lz4               | default         | No       | 
+| snappy            | default         | No       |
+| lzo               | default         | No       |
+| xz                | default         | No       |
+| zstd              | default         | No       |
+| openssl           | default         | No       |
+| protobuf          | 21 (exact)      | Yes      |
```

Comment:
This 'exact' requirement seems to be mostly motivated by considerations for compatibility for presto right?

Reply:
I removed the Protbuf dependency in Presto https://github.com/prestodb/presto/pull/20370
We have to reevaluate this requirement.

Reply:
Ah nice!